website/docs/api/entityruler.md

```diff
@@ -210,6 +210,24 @@ of dicts) or a phrase pattern (string). For more details, see the usage guide on
 | ---------- | ---------------------------------------------------------------- |
 | `patterns` | The patterns to add. ~~List[Dict[str, Union[str, List[dict]]]]~~ |
 
+
+## EntityRuler.remove {#remove tag="method"}
+
+Remove a pattern by its ID from the entity ruler. An `Error` is raised if ID does not exist.
```

Comment:
This should be `ValueError`, right?

Reply:
Yes, I think I made a tini-mini typo, thanks for pointing out!